extra/changelog-generator/changelog-generator

```diff
@@ -167,9 +241,18 @@ if args.all:
         .decode()
         .strip()
     )
-    repos += [os.path.join(base_dir, repo) for repo in repo_list.split()]
+    repos += repo_list.split()
 else:
-    repos.append(".")
+    repos.append(os.path.basename(os.path.realpath(".")))
+
+if not args.base_dir and not args.repo and not args.query_github:
```

Comment:
Just fine, we could play with:
```
not any([args.base_dir, args.repo, args.query_github])
```
But feel free to ignore. It's `_Pythonic_`, but  whether it's _actually_ more readable... 

You be the judge :smile_cat:

Reply:
It *is* more readable! I will change this.